### PR TITLE
Gabe database branch

### PR DESCRIPTION
**Pull request by GabeSArn**
Users are now able to sign in.  

- Once signed in, create page that allows users to upload files and view what else has been uploaded.

- Need to add upload pics functionality

- Working on users logging into their account.

### CODE REVIEW

**Comment by GabeSArn** [comment]
@wbobeirne Please review if able.

File: models/comments.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 const Sequelize = require("sequelize");
-const sql = require("../utility/sql");
+const sql = require("../util/sql.js");
 const User = require("./users.js");
+const bodyParser = require('body-parser');
```
Comment (wbobeirne) [inline models/comments.js]: There's no need to include body-parser in this file.

File: app.js [hunk 2]
```diff
@@ -42,7 +43,8 @@ app.get("/signup", function(req,res) {
 });
 
 app.post("/signup", function(req,res) {
-	res.send("GO");
+	Users.signup(req);
+	res.send("Uploaded to database.");
```
Comment (wbobeirne) [inline app.js]: How do you know signup was successful? It returns a promise, so you should wait for signup to be done before saying it went okay, or let them know it went wrong if signup had an error.

File: models/likes.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 const Sequelize = require("sequelize");
-const sql = require("../utility/sql");
+const sql = require("../util/sql.js");
 const User = require("./users.js");
 const Photos = require("./photos.js");
+const bodyParser = require('body-parser');
```
Comment (wbobeirne) [inline models/likes.js]: There's no need to include body-parser in this file. (You don't use it anywhere in here.)

File: models/photos.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
 const Sequelize = require("sequelize");
-const sql = require("../utility/sql");
+const sql = require("../util/sql.js");
 const User = require("./users.js");
 const Likes = require("./likes.js");
+const bodyParser = require('body-parser');
```
Comment (wbobeirne) [inline models/photos.js]: Same as before.

File: models/users.js [hunk 5]
```diff
@@ -1,20 +1,22 @@
 const Sequelize = require("sequelize");
-const sql = require("../utility/sql");
+const sql = require("../util/sql.js");
 const Photos = require("./photos.js");
 const Comments = require("./comments.js");
 const Jimp = require("jimp"); // An image processing library for Node written entirely in JavaScript
 const bcrypt = require("bcrypt"); // Password hashing
 const fs = require("fs-extra"); // Adds extra file system methods
-
+const bodyParser = require('body-parser');
```
Comment (wbobeirne) [inline models/users.js]: Same as before

File: models/comments.js [hunk 6]
```diff
@@ -1,12 +1,14 @@
...
 
 
 const Comments = sql.define("comments", {
 	id: {
 		type: Sequelize.INTEGER,
 		notNull: true,
+		primaryKey: true,
```
Comment (wbobeirne) [inline models/comments.js]: How is the ID determined? Right now you'd have to manually assign each comment an ID.

**Pull request merged**